src/status_im/contexts/wallet/send/transaction_settings/gas_amount/view.cljs

```diff
@@ -5,26 +5,46 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
-(defn gas-amount-hint-text
-  [{:keys [current]} lower-limit-exceeded? _upper-limit-exceeded?]
+(defn hint-and-status
+  [gas-amount-from-route {:keys [low high]} entered-value]
   (cond
-    lower-limit-exceeded? (i18n/label :t/gas-amount-lower {:current current})
-    :else                 (i18n/label :t/current-units {:current current})))
+    (> entered-value high)                          {:hint-text (i18n/label :t/gas-amount-higher
+                                                                            {:high high})
+                                                     :status    :error}
+    (> entered-value (* 1.1 gas-amount-from-route)) {:hint-text (i18n/label :t/gas-amount-higher-than
+                                                                            {:current
+                                                                             gas-amount-from-route})
+                                                     :status    :warning}
+
+    (< entered-value low)                           {:hint-text (i18n/label :t/gas-amount-lower
+                                                                            {:low low})
+                                                     :status    :error}
+    (< entered-value (* 0.9 gas-amount-from-route)) {:hint-text (i18n/label :t/gas-amount-lower-than
+                                                                            {:current
+                                                                             gas-amount-from-route})
+                                                     :status    :warning}
+    :else                                           {:hint-text (i18n/label :t/current-units
+                                                                            {:current
+                                                                             gas-amount-from-route})
+                                                     :status    :default}))
 
 (defn view
   []
-  (let [suggested-values (rf/sub [:wallet/tx-settings-max-gas-amount])
-        hint-text        (partial gas-amount-hint-text suggested-values)]
+  (let [spectrum              {:low  21000
```

Comment:
IMO `spectrum` implies values in between. I think `gas-boundaries` might be more self-explanatory.